docs/topics/proc_exposing-application-metrics-using-prometheus-with-spring-boot.adoc

```diff
@@ -0,0 +1,175 @@
+
+[id='exposing-metrics-using-prometheus-with-spring-boot_{context}']
+= Exposing application metrics using Prometheus with {SpringBoot}
+
+In this example, you:
+
+* Configure your application to expose metrics.
+* Collect and view the data using Prometheus.
+
+Prometheus actively connects to a monitored application to collect data; the application does not actively send metrics to a server.
+
+.Prerequisites
+
+* Prometheus configured to collect metrics from the application:
++
+--
+. Download and extract the link:https://prometheus.io/download/[archive^] with the latest Prometheus release:
++
+[source,bash,options="nowrap"]
+----
+$ wget https://github.com/prometheus/prometheus/releases/download/v2.4.3/prometheus-2.4.3.linux-amd64.tar.gz
+$ tar -xvf  prometheus-2.4.3.linux-amd64.tar.gz
+----
+
+. Navigate to the directory with Prometheus:
++
+[source,bash,options="nowrap"]
+----
+$ cd  prometheus-2.4.3.linux-amd64
+----
+
+. Append the following snippet to the `prometheus.yml` file to specify the port where your application exposes the metrics endpoint.
++
+The `scrape_interval` setting is optional and sets Prometheus to collect metrics every 5 seconds.
++
+[source,yaml,options="nowrap"]
+----
+  - job_name: 'spring-actuator'
+    metrics_path: '/prometheus'
+    scrape_interval: 5s
+    static_configs:
+    - targets: ['localhost:8080']
```

Comment:
As the project will be deployed within OpenShift, then you must add a remark about how to get the OpenShift route in order to change the target - Make sense @inoxx03 ?

Reply:
@cmoulliard In this example, we do not take OpenShift into account, and the entire deployment happens locally. When the necessary bits are in OpenShift, we will adapt this procedure to document that use case.